### ISSUE TITLE
5GHz Wifi analizer empty on TG789VAC

### ISSUE THREAD

**Issue by boynet7000** [Ansuel/tch-nginx-gui#878]
Device Model: Technicolor TG789VAC v2
Gui Version: 9.5.4

Description of problem: 5GHz wifi analyzer doesn't find anything and stays empty, but 2.4 analyzer works well. Both 2.4 and 5 GHz access points work well.

How to reproduce: via webUI, go to "wireless", analyze 5GHz and confirm to scan

**Comment by FrancYescO** [comment]
firmware?

**Comment by boynet7000** [comment]
version Mint 17.2.0278
same problem on GUI 9.5.8

**Comment by boynet7000** [comment]
firmware mint 17.2
same problem on GUI 9.5.8

**Comment by FrancYescO** [comment]
The spinner will stay forever or the graphs are just empty?
On 17.2 I cannot reproduce (is just working) on tiscali firmware 16.3 the spinner will never stop loading..

**Comment by boynet7000** [comment]
graphs and wlan list empty
![Cattura](https://user-images.githubusercontent.com/47531173/71512079-fb9f1880-2894-11ea-908e-61cfda2d99e0.JPG)

**Comment by FrancYescO** [comment]
Still haven't find a way to reproduce your behaviour.
are you connected to 5ghz while scanning?
post a `logread | grep error` after a scan
check if browser console (press F12) show up errors

**Comment by boynet7000** [comment]
>are you connected to 5ghz while scanning?
no, connected via LAN. It is also warned that all connections are loste while scanning

>do you have both 5ghz and 2.4 AP active?
yes

>any guest AP?
no, disactivated

Just updated to GUI version 5.28.

![analisi_pagina](https://user-images.githubusercontent.com/47531173/75632109-3c027600-5bf9-11ea-917c-c449e2003ad1.JPG)
![Console](https://user-images.githubusercontent.com/47531173/75632110-3d33a300-5bf9-11ea-9250-daec0057a39c.JPG)

>post a logread | grep error after a scan
Sun Mar  1 20:10:35 2020 daemon.err nginx: 2020/03/01 20:10:35 [error] 20523#0:                               *6 attempt to set ngx.header.HEADER after sending out response headers, client:                               192.168.1.10, server: localhost, request: "POST / HTTP/1.1", host: "192.168.1.3"                              , referrer: "http://192.168.1.3/"
Sun Mar  1 20:10:42 2020 daemon.err transformer[20029]: [truncated] getter(rpc.n                              etwork.interface.@., dnsservers) threw an error: ...share/transformer/mappings/r                              pc/network.interface.map:672: bad argument #1 to 'concat' (table expected, got s                              tring)

hope this helps...
remember that whit older GUI version this didn't happened

**Comment by FrancYescO** [comment]
Did you manually edited network file to set dns?

You have to select the console tab after pressing F12

try sharing /etc/config/network and wireless files (replace passwords..)

Btw i'm pretty sure is something wrong in you config as i've tried on multiple device and is working flawlessly

**Comment by boynet7000** [comment]
> Did you manually edited network file to set dns?

no... I'm not so skilled... all config made through GUI
[network.txt](https://github.com/Ansuel/tch-nginx-gui/files/4275154/network.txt)


> You have to select the console tab after pressing F12

It is in the second print screen, isn't it?
 
> try sharing /etc/config/network and wireless files (replace passwords..)
[network.txt](https://github.com/Ansuel/tch-nginx-gui/files/4275157/network.txt)
[wireless.txt](https://github.com/Ansuel/tch-nginx-gui/files/4275177/wireless.txt)


> Btw i'm pretty sure is something wrong in you config as i've tried on multiple device and is 
> working flawlessly

config never changed after I (hardly) found a working set up months ago. Updating GUI to not_remember_what_version created this problem...

**Comment by FrancYescO** [comment]
Try executing `/etc/init.d/transformer restart` than again the `logread | grep error` command

**Comment by boynet7000** [comment]
done: trasformer restart, analize5ghz, logread
here it is:
Tue Mar  3 12:36:07 2020 daemon.err nginx: 2020/03/03 12:36:07 [error] 4231#0: *2 [lua] sessionmgr.lua:563: handleAuth(): Failed logon attempt for user mirco, client: 192.168.1.10, server: localhost, request: "POST /authenticate HTTP/1.1", host: "192.168.1.3", referrer: "http://192.168.1.3/"
Tue Mar  3 12:36:17 2020 daemon.err transformer[3244]: [truncated] getter(rpc.network.interface.@., dnsservers) threw an error: ...share/transformer/mappings/rpc/network.interface.map:672: bad argument #1 to 'concat' (table expected, got string)
